srcIdl/perftest.idl

```diff
@@ -80,8 +82,172 @@ struct TestDataKeyedLarge_t {
     long latency_ping;
     sequence<octet> bin_data;
   #ifdef RTI_CUSTOM_TYPE
-    RTI_CUSTOM_TYPE custom_type; //@key
+    @key
+    RTI_CUSTOM_TYPE custom_type;
     long custom_type_size;
   #endif
 }; //@Extensibility FINAL_EXTENSIBILITY
 
+
+#ifdef RTI_FLATDATA_AVAILABLE
+/*
+ * FLATDATA
+ */
+@mutable
+@language_binding(FLAT_DATA)
+struct TestData_FlatData_t {
+    octet key[KEY_SIZE];
+
```

Comment:
unnecessary new line in all FlatData types in this file

Reply:
Done